fluent/migrate/transforms.py

```diff
@@ -277,43 +277,16 @@ def createVariant(zipped_enum):
 
 
 class CONCAT(Transform):
-    """Concatenate elements of many patterns."""
+    """Create a new Pattern from Patterns, PatternElements and Expressions."""
 
-    def __init__(self, *patterns):
-        self.patterns = list(patterns)
+    def __init__(self, *elements):
+        self.elements = list(elements)
```

Comment:
I wonder if you could just do

    def __init__(self, *arg_elements, elements=None):
        if elements is None:
            elements = arg_elements
        self.elements = list(elements)

and then you wouldn't need `get_init_args` at all?

Reply:
Ah, good idea! `*varargs` must follow all keyword args but something like the following should work:

```py
def __init__(self, *elements, **kwargs):
    kwelements = kwargs.get('elements', None)
    if kwelements is not None:
        self.elements = list(kwelements )
    else:
        self.elements = list(elements)
```